controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -126,14 +126,17 @@ controls:
       levels: level_2
   - id: '5.5'
     title: Extensible Admission Control
-    status: pending
+    status: automated
     rules: []
     controls:
     - id: 5.5.1
       title: Configure Image Provenance using image controller configuration parameters
-      status: pending
-      rules: []
-      levels: level_2
+      status: automated
+      rules:
+        - ocp_allowed_registries
+        - ocp_allowed_registries_for_import
+        - ocp_insecure_registries
```

Comment:
I'd say yes, so that we lean towards more secure posture. But I don't know how common it is for users to wish for insecure registries.

If users really need to use insecure registries, we could have a rule that checks whether the insecure sources match the values configured in a variable. This would require the admin to customize the profile and add the allowed insecure Registries, though.

Reply:
Ok - I'll include it for now and we can smooth out the use case for making it easier to use insecure registries after (seems like an odd case though).

Reply:
@rhmdnd Don't forget the `cis` reference in `ocp_insecure_allowed_registries_for_import`

Reply:
Aha - good catch.